lib/hammer_cli/ca_cert_fetcher.rb

```diff
@@ -1,39 +1,63 @@
+require 'hammer_cli/ca_cert_manager'
 module HammerCLI
   class CACertFetcher
-    def self.fetch_ca_cert(host)
-      CACertFetcher.new.fetch_ca_cert(host)
-    end
-
-    def fetch_ca_cert(host)
+    def fetch_ca_cert(service_uri, ca_path)
       begin
-        uri = URI.parse(host)
+        uri = URI.parse(service_uri)
+        raise URI::InvalidURIError.new(scheme_error(uri)) unless uri.scheme == 'https'
+        ca_cert_manager = HammerCLI::CACertManager.new(ca_path)
+        ca_cert_manager.add_ca_cert(uri)
 
-        hostname = uri.host
-        port = uri.port || 443
+        rh_install_path = "/etc/pki/ca-trust/source/anchors/"
+        rh_update_cmd = "update-ca-trust"
+        deb_install_path = "/usr/local/share/ca-certificates/"
+        deb_update_cmd = "update-ca-certificates"
+        cert_file = ca_cert_manager.cert_file_name(uri)
 
-        if hostname.nil?
-          $stderr.puts _("Couldn't parse URI '%s'.") % host
-          $stderr.puts scheme_error(uri)
-          return HammerCLI::EX_SOFTWARE
-        end
+        puts _("CA certificate for #{service_uri} was stored to #{cert_file}")
+        puts _("Now hammer can use the downloaded certificate to verify SSL connection to the server.")
+        puts
+        puts _("Be aware that hammer cannot verify whether the certificate is correct and you should verify its authenticity.")
+        puts
+        puts _("You can display the certificate content with")
+        puts "  $ openssl x509 -text -in #{cert_file}"
+        puts
 
-        puts get_ca_cert(hostname, port)
+        cert_install_msg = "\n" + _("As root you can also install the certificate and update the system-wide list of trusted CA certificates as follows:") + "\n"
```

Comment:
That results in printing two newlines between the `openssl` command and "As root you can also install...". Was that intentional? (just to make sure)